src/Network/Algorand/Api.hs

```diff
@@ -78,10 +78,14 @@ data NodeApi route = NodeApi
   } deriving (Generic)
 
 -- | Algod Indexer API
-newtype IndexerApi route = IndexerApi
+data IndexerApi route = IndexerApi
   { _accountIdx :: route
       :- "accounts"
       :> Capture "address" Address
       :> QueryParam "round" Round
       :> Get '[JSON] IdxAccountResponse
+  , _blockIdx :: route
+      :- "blocks"
+      :> Capture "round" Round
+      :> Get '[JSON] Block
```

Comment:
Does it not accept format? Msgpack should be more efficient to process, so it'd be nice to use it instead of JSON if possible.

Reply:
Hm, can you check whether passing `format=msgpack` works? Maybe it's just missing in documentation. If it doesn't work, can you please ask in #stakerdao-serokell-randlabs whether they will support it (and say that we'd like to use it because it's more efficient)?

Reply:
```
{
  "message": "Unknown parameter detected: format"
}
```

Reply:
> Mauro Leggieri  ahhh, no, the algorand indexer and our only support json

Reply:
Okay, that's quite sad. I was told by @kirelagin that our JSON parsers are fragile and we should test them carefully, they may work incorrectly in some cases. I'll take a deeper look into that.

Reply:
okay